src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
+
+  if (days < 3) {
+    return result;
+  } else if (days >= 3 && days < 7) {
+    return result - 20;
```

Comment:
5. [Checklist] DON'T add `else` after `if` with return - the code after it won't be executed anyway.